Northstar.CustomServers/mod/scripts/vscripts/mp/_stats.nut

```diff
@@ -1035,7 +1035,11 @@ void function SaveStatsPeriodically_Threaded()
 	while( true )
 	{
 		foreach( entity player in GetPlayerArray() )
+		{
+			if ( !IsValid( player ) )
+				continue
 			Stats_SaveAllStats( player )
```

Comment:
why not
`if( IsValid( player ) ) Stats_SaveAllStats( player )`

Reply:
I'd prefer this over a guard clause in this case, since we are just doing one thing
```suggestion
		{
			if ( IsValid( player ) )
				Stats_SaveAllStats( player )
```